runtime/rasdump/dmpmap.c

```diff
@@ -70,7 +70,14 @@ static const DgDumpAction dgActions[] = {
 
 static const int numDumpActions = ( sizeof(dgActions) / sizeof(DgDumpAction) );
 
+/* these are the filter strings used for the default OutOfMemory dump options */
+#define  SYSTHROW_OOM_1_1 "events=systhrow,range=1..1,filter=java/lang/OutOfMemoryError,request=exclusive+compact+prepwalk"
```

Comment:
`SYSTHROW_OOM_1_1` is new.  I added the macro for consistency.

Reply:
I'd rather remove it unless it's used.  Otherwise, the next reader of the code may spend time searching for its use or wonder (like I did) whether there was a bug in the code below where one of the  SYSTHROW_OOM_1_4  might have been a 1_1

Reply:
Okay.  Will do.